### PR TITLE
103820474 Review and rate events

### PR DESCRIPTION
**Pull request by andela-gogbara**
#### What does this PR do?
It gives users attending an event the ability to review and rate the event after the event has ended.
#### Description of Task to be completed?
Added review model and controller
Added feature test for reviews
Added controller test for review controller
#### Any background context you want to provide?
This feature was initially was supposed to be commenting and rating of an event but was changed to review and rating of an event. A story has been added to pivotal tracker to implement the commenting feature. 
#### What are the relevant pivotal tracker stories?
#103820474

### CODE REVIEW

File: app/assets/javascripts/review.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      empty_fields();
+    }).fail(function () {
+      Materialize.toast('Sorry, review cannot be saved', 3000);
+    });
+    event.preventDefault();
+  });
+  function filled_in_stars(number, target_div) {
+    var stars = '';
+    for (var i = 1; i <= number; i++) {
+      stars += '<span class=\'filled-in-star star-size\'>&#9734;</span>';
```
Comment (andela-sdamian) [inline app/assets/javascripts/review.js]: you can add the html string to a variable 

```var stars = '',

File: app/assets/javascripts/review.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    }
+    target_div.find('.review-rating').html(stars);
+  }
+  function empty_stars(number, target_div) {
+    var stars = '';
+    var count = 5 - number;
+    for (var i = 1; i <= count; i++) {
+      stars += '<span class=\'star-size\'>&#9734;</span>';
```
Comment (andela-sdamian) [inline app/assets/javascripts/review.js]: same feedback as above

File: app/assets/javascripts/review.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    }
+    target_div.find('.review-rating').append(stars);
+  }
+  function add_author_image(target_div) {
+    var image_source = $('img.profile_pic').attr('src');
+    var img = target_div.find('.review-author-pic');
```
Comment (andela-sdamian) [inline app/assets/javascripts/review.js]: use single `var`, replace all instance of this.

File: app/helpers/events_helper.rb [hunk 4]
```diff
@@ -40,6 +40,14 @@ def event_price(tickets, event)
...
   end
 
+  def past_event?(event)
+    if Time.now > event.end_date
+      true
+    else
+      false
+    end
+  end
+
```
Comment (andela-sdamian) [inline app/helpers/events_helper.rb]: this method can be as simple as 

```
def past_event?(event) 
   Time.now > event.end_date 
end 
```

File: app/assets/stylesheets/reviews.scss [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+.star-size {
+  font-size: 1em;
+}
+
+.event-star-size {
+  font-size: 3em;
+}
+
+.event-filled-in-star::before {
+  // content: "\2605";
```
Comment (andela-bmakinwa) [inline app/assets/stylesheets/reviews.scss]: **Nitpick - Code Quality**
Remove commented out code

**Pull request merged**